app/views/support_interface/application_forms/application_choices/confirm_revert_rejection.html.erb

```diff
@@ -14,7 +14,6 @@
 
       <p class="govuk-body">A rejection can only be reverted if:</p>
       <ul class="govuk-list govuk-list--bullet">
-        <li>it was rejected within 5 working days of the request to revert</li>
         <li>the candidate has not accepted any other offers</li>
       </ul>
```

Comment:
Can we reword this into a single sentence please? Don't think we need a bullet list if there’s just one item in it.